LOG: ====================================
LOG: Benchmarking HyCC
LOG: ====================================
LOG: TEST PATH: /home/ubuntu/ejchen/circ_benchmarks/modules/HyCC/examples/benchmarks/kmeans/kmeans.c
LOG: SPEC_FILE: /home/ubuntu/ejchen/circ_benchmarks/specs/kmeans.spec
LOG: MINIMIZATION TIME: 600
LOG: COST MODEL: hycc
LOG: Running with args: ['--all-variants']
LOG: ====================================
LOG: Test: Build circuit time
LOG: Compile time: 2425.12
LOG: Time / Memory: 2425.74 seconds 25929904 kB
LOG: ====================================
LOG: Test: Module bundle time
LOG: Time / Memory: 0.03 seconds 10104 kB
LOG: ====================================
LOG: Test: RERUN 0: yaohybrid
LOG: Error: Process returned with status code 134
LOG: Error message: terminate called after throwing an instance of 'std::out_of_range'   what():  _Map_base::at Command terminated by signal 6 26.65 seconds 3809704 kB
LOG: ====================================
LOG: Test: RERUN 0: gmwhybrid
LOG: Error: Process returned with status code 134
LOG: Error message: terminate called after throwing an instance of 'std::out_of_range'   what():  _Map_base::at Command terminated by signal 6 27.19 seconds 4023352 kB
LOG: ====================================
LOG: Running with args: ['--all-variants', '--outline']
LOG: ====================================
LOG: Test: Build circuit time
LOG: Error: Process returned with status code 134
LOG: Error message: <command-line>: warning: "__STDC_VERSION__" redefined <built-in>: note: this is the location of the previous definition <command-line>: warning: "__STDC_VERSION__" redefined <built-in>: note: this is the location of the previous definition terminate called after throwing an instance of 'std::runtime_error'   what():  No reaching definitions analysis result available Command terminated by signal 6 1.22 seconds 70784 kB
LOG: Failed building circuit with args: ['--all-variants', '--outline'], exception:
LOG: ====================================
LOG: Test: RERUN 0: yaohybrid
LOG: Error: Process returned with status code 134
LOG: Error message: terminate called after throwing an instance of 'std::out_of_range'   what():  _Map_base::at Command terminated by signal 6 27.83 seconds 3809820 kB
LOG: ====================================
LOG: Test: RERUN 0: gmwhybrid
LOG: Error: Process returned with status code 134
LOG: Error message: terminate called after throwing an instance of 'std::out_of_range'   what():  _Map_base::at Command terminated by signal 6 28.57 seconds 4023120 kB
LOG: ====================================
